deployability/modules/testing/tests/test_manager/test_uninstall.py

```diff
@@ -51,8 +52,8 @@ def test_uninstall(wazuh_params):
         manager_status = GeneralComponentActions.get_component_status(manager, 'wazuh-manager')
         assert 'active' in manager_status, logger.error(f'The {HostInformation.get_os_name_and_version_from_inventory(manager)} is not active')
     for _, manager_params in wazuh_params['managers'].items():
-        WazuhManager.perform_uninstall_and_scan_for_manager(manager_params)
-
+        #WazuhManager.perform_uninstall_and_scan_for_manager(manager_params)
```

Comment:
Remove comment

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5324/commits/96c47257f1bca33c7f144285a4c7b63a8c10fbce